README.md

```diff
@@ -90,6 +90,54 @@ of the analysis. It should follow the format of JSON-schema
 
 Output types are described further [below](#output-types).
 
+## Input Form Layout Endpoint (optional)
+
+Endpoint `/layout/inputs/<task-name>` may provide simplistic specification
+of a form's layout needed to gather computational task input.
+
+Sample JSON for such styling:
+
+```
+[
+    {
+        "type": "section",
+        "title": "Personal details",
+        "items": ["firstName", "lastName", "age"],
+        "expandable": true,
+        "expanded": false,
+    },
+    ... // another elements of the layout
+]
+```
+
+Format of `layout` should be as samples included [here](https://angular2-json-schema-form.firebaseapp.com/).
+
+Expected HTTP return code is `200`. If task of the given name does not exist,
+return code is `404`. Supported HTTP method is `GET`.
+
+
+## Output Form Layout Endpoint
+
+Endpoint `/layout/outputs/<task-name>` should provide specification of a
```

Comment:
What should be put in task-name field in order to get proper data from api?

Reply:
I've extended the doc.